### ISSUE TITLE
value and policy targets are generated with 'params_target' in paper, but 'params' in code

### ISSUE THREAD

**Issue by baskuit** [deepmind/open_spiel#967]
See the loss function in the repo:
https://github.com/deepmind/open_spiel/blob/master/open_spiel/python/algorithms/rnad/rnad.py#L777

`      v_target_, has_played, policy_target_ = v_trace(
          v_target,
          ts.env.valid,
          ts.env.player_id,
          ts.actor.policy,
          policy_pprocessed,`

`policy_pprocessed = self.config.finetune(pi, ts.env.legal, learner_steps)`

`pi, v, log_pi, logit = rollout(params, ts.env)`

It seems to be using params, but the paper says params_target:

![image](https://user-images.githubusercontent.com/77259253/204002229-302643c1-b390-4ed7-8444-9f0f25899544.png)

![image](https://user-images.githubusercontent.com/77259253/204002251-ec06fc9a-7ce1-4905-a60f-c230cd380cb9.png)

**Comment by lanctot** [comment]
@perolat can you take a look?

**Comment by spktrm** [comment]
I also noticed in this line:
```python
    log_policy_reg = log_pi - (alpha * log_pi_prev + (1 - alpha) * log_pi_prev_)
```
It seems the policy regularisation also uses params instead of params target

**Comment by perolat** [comment]
Hi @baskuit,

Thanks a lot for going in the details of the paper!

This question should be addressed with an updated version of the appendix that will be public soon. I'll add the link once it is  public.

In short we use \theta_n for the policy and \theta_{n, target} for the value (what the code does). Sorry for the discrepancy between the code and the text.

Julien

**Comment by lanctot** [comment]
Thanks for the clarification @perolat !
